c2rust-transpile/src/translator/mod.rs

```diff
@@ -3177,11 +3183,10 @@ impl<'c> Translation<'c> {
     }
 
     fn compute_size_of_ty(&self, ty: Box<Type>) -> TranslationResult<WithStmts<Box<Expr>>> {
-        let std_or_core = if self.tcfg.emit_no_std { "core" } else { "std" };
         let name = "size_of";
         let params = mk().angle_bracketed_args(vec![ty]);
         let path = vec![
-            mk().path_segment(std_or_core),
+            mk().path_segment(self.std_or_core()),
             mk().path_segment("mem"),
             mk().path_segment_with_args(name, params),
         ];
```

Comment:
This can also be `Vec<&str>` (without `mk().path_segment` calls).

Reply:
It can't because `mk().path_segment_with_args(name, params)` is `PathSegment`.